_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
This is the only file that should be in this PR. All the others should not be part of this PR. I am not sure how they are all showing up as changed, since they all existed in the template that you forked from.

Also, you need to rename the post to match the date and title of the post, currently is it still 'welcome-to-jekyll'